open-craft/demo-mfe
# [STAR-2442] - Rewrite state management with Providence

This PR removes the extended and verbose state management code and replaces it with Providence. The result has full feature parity but a fraction of the lines and a great deal more clarity.

**JIRA tickets**: https://tasks.opencraft.com/browse/STAR-2442

**Testing instructions**:

1. Have a working and running devstack
2. npm install
3. npm run start
4. Visit https://localhost:8080/catalog/ and verify things are working correctly.
